pubtools/pulplib/_impl/model/maintenance.py

```diff
@@ -219,4 +219,9 @@ def remove(self, repo_ids, **kwargs):
             if entry.repo_id not in repo_ids:
                 new_entries.append(entry)
 
-        return attr.evolve(self, last_updated_by=owner, entries=new_entries)
+        return attr.evolve(
+            self,
+            entries=new_entries,
+            last_updated_by=owner,
+            last_updated=datetime.datetime.utcnow(),
```

Comment:
Could you explain this part a little?  The change here was to add/overwrite last_updated on any change.  It might be correct, but all of the commit message, PR message and changelog only refer to changes to last_updated_by, a different field.

Reply:
Hm, I thought the problem was here for the first time and did this change, but it's not actually. 
I'll remove this since it's unrelated change